cpu/nrf52/radio/nrf802154/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig KCONFIG_MODULE_NRF802154
+    bool    "Configure nRF802154"
+    depends on MODULE_NRF802154
+    help
+        Configure nRF802154 module using Kconfig.
+
+if KCONFIG_MODULE_NRF802154
+
+config NRF802154_CCA_THRESH_DEFAULT
+    int "Default CCA threshold value"
+    default -90
```

Comment:
Shouldn't this be `0x14` like [here](https://github.com/RIOT-OS/RIOT/pull/13145/files#diff-2f9d72eea34c3a7a53ebd3709ca98b88R58)?

Reply:
exactly. To be fixed

Reply:
done